test/Microsoft.AspNet.Mvc.TagHelpers.Test/AnchorTagHelperTest.cs

```diff
@@ -34,17 +35,19 @@ public class AnchorTagHelperTest
                 },
                 items: new Dictionary<object, object>(),
                 uniqueId: "test",
-                getChildContentAsync: () => Task.FromResult("Something Else"));
+                getChildContentAsync: () => {
```

Comment:
nit: leaving a trailing curly brace on a line is a bad habit; something to avoid even in test code.